node-manager/src/nodemanager.rs

```diff
@@ -114,6 +114,16 @@ impl NodeManager {
             .get_total()
     }
 
+    #[wasm_bindgen]
+    pub async fn send_to_address(&self, destination_address: String, amount: u64) -> String {
```

Comment:
should make fee an optional param here as well, and if it is `None` we can get the free rate from esplora